docker/docker_image_dest.go

```diff
@@ -8,31 +8,33 @@ import (
 	"net/http"
 
 	"github.com/Sirupsen/logrus"
-	"github.com/containers/image/docker/reference"
 	"github.com/containers/image/manifest"
 	"github.com/containers/image/types"
 )
 
 type dockerImageDestination struct {
-	ref reference.Named
-	tag string
-	c   *dockerClient
+	dockerReference
```

Comment:
Using unnamed members like this now to minimize clutter in the patch, I guess this should use a real named member in the future for clarity.